### PR TITLE
Added an image when no profiles are found

### PR DESCRIPTION
**Pull request by andriiwp3**
Added an Omega icon component and inserted below the text when no profiles were found

### CODE REVIEW

File: src/pages/profiles/profiles.page.js [hunk 1]
```diff
@@ -44,7 +45,10 @@ const ProfilesPage = () => {
       <SearchContainer value={searchTerm} onChange={handleChange} />
       <>
         {!data?.results.length ? (
-          <h3 className="title">Схоже таких профілів не знайдено</h3>
+			<>
+				<h3 className="title">Схоже таких профілів не знайдено</h3>
+				<OmegaIcon className="not-found-icon" />
+			</>
```
Comment (AndriyTykhan) [inline src/pages/profiles/profiles.page.js:51]: Look like you have no linter setup. Could you please do it and format the code correctly?

**Comment by AndriyTykhan** [comment]
And could you please add a screenshots of your work?

**Comment by andriiwp3** [comment]
Yes, take it)
image

And mobile version
image

**Comment by andriiwp3** [comment]
So, you're right. I didn't even notice that my global settings override the linter configuration of the project

**Review by AndriyTykhan — APPROVED** [review]

**Comment by AndriyTykhan** [comment]
Looks great! Feel free to merge

**Pull request merged**